modules/user/user.service.js

```diff
@@ -15,7 +15,16 @@ class UserService {
     });
 
     if (checkedUser) {
-      const massage = 'User with provided email already exists';
+      const massage = [
```

Comment:
Please, uppercase this error message like it is done for error messages with `*_NOT_FOUND` and move it to the top of the file

Reply:
fixed